src/content-helper/dashboard-widget/class-dashboard-widget.php

```diff
@@ -59,29 +122,28 @@ public function add_dashboard_widget(): void {
 	 * Enqueues the Dashboard Widget's assets.
 	 *
 	 * @since 3.7.0
-	 *
-	 * @param string $hook_suffix The current admin page.
 	 */
-	public function enqueue_assets( $hook_suffix ): void {
-		if ( 'index.php' === $hook_suffix ) {
-			$asset_php        = get_asset_info( 'build/content-helper/dashboard-widget.asset.php' );
-			$built_assets_url = plugin_dir_url( PARSELY_FILE ) . 'build/content-helper/';
-
-			wp_enqueue_script(
-				'wp-parsely-dashboard-widget',
-				$built_assets_url . 'dashboard-widget.js',
-				$asset_php['dependencies'],
-				$asset_php['version'],
-				true
-			);
-
-			wp_enqueue_style(
-				'wp-parsely-dashboard-widget',
-				$built_assets_url . 'dashboard-widget.css',
-				array(),
-				$asset_php['version']
-			);
+	public function enqueue_assets(): void {
+		if ( ! $this->should_be_enabled() ) {
```

Comment:
We are using this function in two place i.e. inside each callback of the action ..... so can we just use it once inside `run` function?

Reply:
The reason for this is explained in the [run() function](https://github.com/Parsely/wp-parsely/blob/f5db4851f6958aac2fccf044ba1d8c9aa654be0e/src/content-helper/dashboard-widget/class-dashboard-widget.php#L66-L67), but I see that the function name in the comment is wrong. I'll update it.

Reply:
This is done.